sprout/scripts/config_validation/bgcf_schema.json

```diff
@@ -0,0 +1,53 @@
+{
+  "title": "All BGCF routes",
+  "type": "object",
+  "properties": {
+    "routes": {
+      "type": "array",
+      "items": {
+        "title": "Each BGCF route",
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "domain": {
+            "type": "string",
+            "description": "SIP trunk IP address or domain name"
+          },
+          "number": {
+            "type": "string",
+            "description": "Telephone number",
+            "pattern": "^\\+{0,1}[0-9]*$"
+          },
+          "route": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "format": "uri",
```

Comment:
Can we specify both `"format"` and `"regex"`. If that's possible, we could just set the regex to "^sip:". (Note that just checking for the regex you suggested doesn't actually work as the bit before the `@` doesn't have to be a telephone number)